web/packages/extension/global.d.ts

```diff
@@ -0,0 +1,8 @@
+declare global {
```

Comment:
Isn't this file meant to be in in the folder src, named globals.d.ts? If it is, I don't think the tsconfig.json would need changes.